app/presenters/backup_code_depleted_presenter.rb

```diff
@@ -0,0 +1,40 @@
+class BackupCodeDepletedPresenter
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::TranslationHelper
+
+  def initialize(view)
```

Comment:
Why are you saving the view context here? It looks like it isn't used in the presenter?